recipes/systemc-cci/1.0.0/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class SystemccciConan(ConanFile):
+    name = "systemc-cci"
+    version = "1.0.0"
+    description = """SystemC Configuration, Control and Inspection library"""
+    homepage = "https://www.accellera.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    topics = ("simulation", "modeling", "esl", "cci")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+    requires = "systemc/2.3.3"
+    generators = "make"
+    exports_sources = "patches/**"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+
+        if self.settings.os == "Macos":
+            raise ConanInvalidConfiguration("Macos build not supported")
+
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("The compilation of SystemC CCI "
+                                            "as a DLL on Windows is currently "
+                                            "not supported")
+
+        if tools.valid_min_cppstd(self, "17"):
+            raise ConanInvalidConfiguration(
+                "C++ Standard %s not supported by SystemC CCI" %
+                self.settings.compiler.cppstd)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("cci-{}".format(self.version), self._source_subfolder)
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+        env_build = AutoToolsBuildEnvironment(self)
```

Comment:
**UNTESTED**
```suggestion
        env_build = AutoToolsBuildEnvironment(self)
        env_build.cxx_flags.append("-std=c++{}".format(self.settings.compiler.cppstd if self.settings.compiler.cppstd else 11))
```

Reply:
I dont think this is needed as AutoToolsBuildEnvironment gets the cppstd from conan config

https://github.com/conan-io/conan/blob/63b6a28ab36706144dbdc0ddf40ead09522fecc1/conans/client/build/autotools_environment.py#L63

Reply:
Indeed. I added it because there was an error about not having c++11.

Reply:
That one was cause by the fact that CCI does not support c++98